index.html

```diff
@@ -0,0 +1,147 @@
+            >
+          </li>
+          <li><a href="#social-media-marketing">social-media-marketing</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="hero"></div>
+    <!--All images are now responsive-->
+    <!-- contents class divided into 3 sections -->
+    <div class="content">
+      <section 
+      class="search-engine-optimization"
+      id="search-engine-optimization"
+      >
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          alt="A cup of coffee, a laptop and a notebook placed on a wooden table"
+          class="float-left"
+        />
+
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+
+      <section 
+        class="online-reputation-management"
+        id="online-reputation-management"
+      >
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          alt="A laptop showing a company's reputation increasing"
+          class="float-right"
+        />
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+
+      <section 
+        class="social-media-marketing"
+        id="social-media-marketing"
+      > 
+
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          alt="A group of colleagues exchanging information at a meeting table"
+          class="float-left"
+        />
+
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </div>  
+
+    <!--All images are followed with an ALT tag -->
+    <aside class="benefits">
+      <div class="benefit-lead">
+        <h3>Lead Generation</h3>
+        <img
+          src="./assets/images/lead-generation.png"
+          alt="An image illustrating money being produced"
+        
+        />
+
+        <p>
+          Inbound strategies for lead generation require less work for your
+          business, bringing customers directly to your website.
+        </p>
+      <div>
+
+      <div class="benefit-brand">
```

Comment:
A section tag could be used here instead of a `div`. See [here](https://medium.com/design-code-repository/html-elements-section-vs-div-vs-article-a8c34e6548cf) for more information on section vs div.